stable/bookstack/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.25.2
 description: BookStack is a simple, self-hosted, easy-to-use platform for organising and storing information.
 name: bookstack
-version: 1.0.3
```

Comment:
Not 1.1.3 but 1.1.0

Reply:
You're right, sorry for that :)

Reply:
Actually it's just a fix so 1.0.4 is better. Sorry for that.